website/src/pages/code-insights.tsx

```diff
@@ -0,0 +1,583 @@
+                    <span className="keyword">archived:</span>no <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+        {
+            header: 'Vulnerable log4j versions',
+            description: 'What vulnerable log4j versions are present.',
+            queries: [
+                <>
+                    <span className="keyword">lang:</span>gradle org\.apache\.logging\.log4j['"]
+                    2\.(0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16)(\.[0-9]+){' '}
+                    <span className="keyword">patterntype:</span>regexp <span className="keyword">archived:</span>no{' '}
+                    <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+        {
+            header: 'API keys',
+            description: 'How quickly we notice and remove API keys when they are committed.',
+            queries: [
+                <>
+                    regexMatchingAPIKey <span className="keyword">patterntype:</span>regexp{' '}
+                    <span className="keyword">archived:</span>no <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+    ],
+    codeHealth: [
+        {
+            header: 'TODOs',
+            description: 'How many TODOs are in a specific part of the codebase (or all of it).',
+            queries: [
+                <>
+                    TODO <span className="keyword">archived:</span>no <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+        {
+            header: 'Commits with "revert"',
+            description: 'How frequently there are commits with “revert” in the commit message.',
+            queries: [
+                <>
+                    <span className="keyword">type:</span>commit revert <span className="keyword">archived:</span>no{' '}
+                    <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+        {
+            header: 'Linter override rules',
+            description: 'How many linter override rules exist.',
+            queries: [
+                <>
+                    <span className="keyword">file:</span>\.eslintignore .\n{' '}
+                    <span className="keyword">patterntype:</span>regexp <span className="keyword">archived:</span>no{' '}
+                    <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+        {
+            header: 'Deprecated calls',
+            description: 'How many times deprecated calls are used.',
+            queries: [
+                <>
+                    <span className="keyword">lang:</span>java @deprecated <span className="keyword">archived:</span>no{' '}
+                    <span className="keyword">fork:</span>no
+                </>,
+            ],
+        },
+    ],
+}
+
+const blogListItems = [
```

Comment:
These will be updated when we have the correct assets and blog posts to link to.